product_docs/docs/migration_portal/4/02_supported_platforms.mdx

```diff
@@ -6,7 +6,7 @@ title: "Supported platforms"
 
 <div id="supported_platforms" class="registered_link"></div>
 
-The Migration Portal supports assessment and migration from Oracle 11g, 12c, 18c, and 19c to EDB Postgres Advanced Server 11, 12, 13, or 14. Migration Portal is supported on the following browsers.
+The Migration Portal supports assessment and migration from Oracle 11g, 12c, 18c, and 19c to EDB Postgres Advanced Server 12 and later versions. Migration Portal is supported on the following browsers.
```

Comment:
I would stick to the explicit list, especially where you are already being explicit about Oracle versions.

Reply:
Gotcha, so like this: 
```suggestion
The Migration Portal supports assessment and migration from Oracle 11g, 12c, 18c, and 19c to EDB Postgres Advanced Server 12, 13, 14, 15 and 16. Migration Portal is supported on the following browsers.
```
Should we double-check the versions with someone from product/dev? Or is the UI enough "proof" of the current state of compatibility?

Reply:
11 is EOL, 16 is current. No need to check.

Reply:
You may need an oxford comma after 15 tho. :)